distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1237,6 +1238,26 @@ type Column
         new_name = naming_helper.function_name "ends_with" [self, other]
         run_vectorized_binary_case_text_op self Java_Storage.Maps.ENDS_WITH other case_sensitivity (a -> b -> a.ends_with b case_sensitivity) new_name
 
+    ## GROUP Standard.Base.Text
+       ICON preparation
+       Calulates the text length for each element of the column
+
+       In the Database backends, the default text length method of the
+       particular database is used.
+
+       In the in-memory backend, this will give you the grapheme length of the string.
+
+       > Example
+             import Standard.Examples
+
+             example_text_length =
+                Examples.text_column_1.text_length
+    text_length : Column
+    text_length self =
+        Value_Type.expect_text self <|
+          new_name = naming_helper.function_name "text_length" [self]
+          run_vectorized_unary_op self Java_Storage.Maps.TEXT_LENGTH new_name expected_result_type=Value_Type.Integer
```

Comment:
This looks perfect.

A note that the `expected_result_type` actually only makes sense if a `fallback` is provided. But that's a defect of the pre-existing documentation because it did not clarify that. Also ofc there is no harm in setting it, it is just unused.

Reply:
Good to know thanks. In that case I can make this even cleaner.